referenceframe/frame.go

```diff
@@ -400,13 +439,18 @@ func (rf *rotationalFrame) Name() string {
 }
 
 func (rf *rotationalFrame) MarshalJSON() ([]byte, error) {
```

Comment:
Frame name should be updated to `revoluteFrame`

Reply:
Actually, seeing how pervasive both this AND translationalFrame are, I wouldn't expect you to change this just yet, we can probably come back later and clean that up.

Reply:
Yeah there is some naming mismatch throughout rdk.  We internally call them rotational but they are externally referred to as revolute.  I'm going to make a Product ticket to address this

Reply:
PRODUCT-334 now exists